kolibri/plugins/setup_wizard/assets/src/state/actions/forms.js

```diff
@@ -0,0 +1,23 @@
+function submitDefaultLanguage(store, language) {
+  store.dispatch('SET_LANGUAGE', language);
+}
+function submitFacilityName(store, facilityName) {
+  store.dispatch('SET_FACILITY_NAME', facilityName);
+}
+function submitSuperuserCredentials(store, name, username, password) {
```

Comment:
@rtibbles

Reply:
Hrm? This seems fine, no? Could do it with a destructured parameter, but potato, potato.